packages/object-lists/drizzle.config.ts

```diff
@@ -0,0 +1,15 @@
+import type {Config} from 'drizzle-kit';
+import * as dotenv from 'dotenv';
```

Comment:
Curious: would something like `import dotenv from 'dotenv'` also work (i.e. without the `*`)?